src/Persisters/Entity/JoinedSubclassPersister.php

```diff
@@ -61,8 +61,8 @@ protected function getDiscriminatorColumnTableName(): string
      */
     private function getVersionedClassMetadata(): ClassMetadata
     {
-        if (isset($this->class->fieldMappings[$this->class->versionField]->inherited)) {
-            $definingClassName = $this->class->fieldMappings[$this->class->versionField]->inherited;
+        if (isset($this->class->fieldMappings[$this->class->versionField ?? '']->inherited)) {
```

Comment:
If we take this one as an example, does it happen in a realistic scenario, or would it make sense to trigger a deprecation ourselves in the next major release?

Reply:
I have no idea. 🤷 But it's actually trigger a lot of times in the tests (see https://github.com/xabbuh/orm/actions/runs/17612071318/job/50036129395#step:6:84).